src/components/Signup.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './signup.css';
+
+function Signup() {
+  return (
+    <div>
+      <form>
+        <h3>Login Here</h3>
+        <label htmlFor="username">Username</label>
+        <input type="text" placeholder="Email or Phone" id="username" />
+        <label htmlFor="password">Password</label>
+        <input type="password" placeholder="Password" id="password" />
+        <button type="button">Log In</button>
+      </form>
+    </div>
+
+  );
+}
+
+export default Signup;
```

Comment:
why is the function "signup"

Shouldn't it be Login? Please do let me why you used signup in place login or if not change it for clarity sake